docs/guides/users.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Users"
+---
+
+To get a user object in Timber, you use `Timber::get_user()` and pass the WordPress user ID as an argument.
+
+```php
+$user = Timber::get_user( $user_id );
+```
+
+This function is similar to [`get_userdata()`](https://developer.wordpress.org/reference/functions/get_userdata/) and accepts one argument: a user ID. If you don’t pass in any argument, Timber will use `get_current_user_id()` to work with the currently logged in user.
+
+```php
+$user = Timber::get_user();
+
+// Is the same as…
+
+$user = Timber::get_user( get_current_user_id() );
+```
+
+What you get in return is a [`Timber\User`](https://timber.github.io/docs/reference/timber-user/) object, which is similar to `WP_User`.
+
+## Get user by field
+
+If you don’t have a user ID, you can also get a user by other fields, like `email` or `login` through `Timber::get_user_by()`.
+
+```php
+// Get a user by email.
+$user = Timber::get_user_by( 'email', 'user@example.com' );
+
+// Get a user by login.
+$user = Timber::get_user_by( 'login', 'keanu-reeves' );
+```
+
+## Twig
+
+You can convert user IDs to user objects in Twig using the `get_user()` function.
+
+```twig
+{% set user = get_user(user_id) %}
+```
+
+It also works if you have an array of user IDs that you want to convert to `Timber\User` objects. Use the `get_users()` function.
+
+```twig
+{% for user in get_users(user_ids) %}
+
+{% endfor %}
+```
+
+## Invalid user
+
+If no user can be found with the user ID you provided, the `Timber::get_user()` function will return `null`. With this, you can always check for valid users in a template a simple if statement.
+
+```php
+$user = Timber::get_user( $user_id );
+
+if ( $user ) {
+    // Handle user.
+}
+```
+
+Or in Twig:
+
+```twig
+{% if user %}
+    {{ user.name }}
+{% endif %}
+```
+
+## Login state
+
+Similar to checking for valid users, you can also check whether a user is currently logged in to WordPress. When you don’t provide an ID for `Timber::get_user()`, it will return `null` if no user is currently logged in.
```

Comment:
```suggestion
Similar to checking for valid users, you can also check whether a user is currently logged in to WordPress. When you don’t provide an ID for `Timber::get_user()`, it will return `false` if no user is currently logged in.
```